src/serac/physics/utilities/functional/functional.hpp

```diff
@@ -522,6 +561,20 @@ class Functional<test(trial)> : public mfem::Operator {
    * @brief The gradient object used to implement @p GetGradient
    */
   mutable Gradient grad_;
+
+  /**
+   * @brief The element tangent matrix vector for ComputeElementMatrices() and UpdateAssembledSparseMatrix() calls
```

Comment:
"matrix vector" sounds funny to me, maybe something like

```suggestion
   * @brief storage buffer for element stiffness matrices, used in ComputeElementMatrices() and UpdateAssembledSparseMatrix()
```